docs/usage/flower17/flower17.rst

```diff
@@ -0,0 +1,41 @@
+Flower17 Species Classification
+===============================
+
+This example will show you how to train and test a model on the `FLOWER17` dataset.
+This dataset contains 17 classes of flower species, each having 80 images.
+
+Download the dataset and verify with with ``sha384sum``.
+
+.. literalinclude:: /../examples/flower17/dataset.sh
+
+.. code-block:: console
+
+    17flowers.tgz: OK
+
+Extract the dataset.
+
+.. code-block:: console
+
+    $ tar -xzf 17flowers.tgz
+
+All the images are now in a folder called `jpg`.
+Let's split these images into flower_dataset/train and flower_dataset/test directories,
+each directory containing sub-directories corresponding to the 17 flower classes using split.py.
```

Comment:
```suggestion
each directory containing sub-directories corresponding to the 17 flower classes using
**split.py**
```

Reply:
Actually I wrote the splitting code myself, it's not documented anywhere.